test/Constraints/result_builder_empty_case.swift

```diff
@@ -0,0 +1,41 @@
+// RUN: %target-swift-emit-silgen %s -verify | %FileCheck %s
+
+// Tests for a crash that occurred when the result builder transform encountered
+// an empty case statement.
+protocol V { }
+
+struct EV: V { }
```

Comment:
Empty protocol and struct is really needed ? 

I guess yes because of static funcs returns on `VB`.